core/tests/test-data.ts

```diff
@@ -0,0 +1,13 @@
+import { faker } from '@faker-js/faker';
+
+export const testUser = {
+  emailAddress: process.env.TEST_ACCOUNT_EMAIL || '',
+  password: process.env.TEST_ACCOUNT_PASSWORD || '',
+  streetAddress: faker.location.streetAddress(),
+  firstName: faker.person.firstName(),
+  lastName: faker.person.lastName(),
+  phoneNumber: faker.phone.number(),
+  state: faker.location.state(),
+  city: faker.location.city(),
+  zipCode: '76286',
+};
```

Comment:
I was trying to avoid code duplication by doing this, so we have everything in one place. But I agree that different tests have different needs in test data, so this way is not universal. I didn't want to use the way I've seen in other tests - to declare a variable for each input at the top of the file, having this code fragment duplicated in different files. 
Inline usage indeed seems to be the best option for now. I'll update code to use this approach. When we'll have more tests with various test data used, we could create a better solution for handling this. 
Thanks for your input! ☀️